collector/lib/runtime-control/Config.cpp

```diff
@@ -18,8 +20,99 @@ void Config::Update(const storage::RuntimeFilteringConfiguration& msg) {
   std::unique_lock<std::mutex> lock(mutex_);
 
   config_message_.emplace(msg);
+  ConfigMessageToConfig(msg);
+  std::string cluster = "cluster-1";  // Cluster is always the same, so should probably not use it anywhere. Using it but hardcoding it for now.
+  SetBitMasksForNamespaces(cluster);
```

Comment:
We should just clear all the internal state and wait for feature-subcomponents to come back and ask for container details.